code/game/machinery/atmo_control.dm

```diff
@@ -545,6 +546,11 @@
 
 	add_fingerprint(usr)
 
+	if(href_list["in_set_rate"])
+		var/response=input(usr,"Set new flow rate, in L/s. \[0-[200]\]") as num
+		volume_rate_setting = text2num(response)
+		volume_rate_setting = between(0, volume_rate_setting, 200)
```

Comment:
Why do you use `between` here but `Clamp` in air injector ?

Reply:
I'm not actually sure. They seem to do the same thing, but `Clamp` is used in on-click UIs for all the atmos components, while the atmos control computer used `between` for setting the output pressure, so I simply stayed consistent within the file.

Reply:
Alright, makes sense.